soccer/Robot.hpp

```diff
@@ -386,6 +386,8 @@ class OurRobot : public Robot {
 
     void avoidOpponentRadius(unsigned shell_id, float radius);
 
+    Geometry2d::Point mouthPos() const;
```

Comment:
I don't like the name `mouthPos`, since it's the center of the mouth. Perhaps `mouthCenterPos` or similar? I think a better name can be thought up though...